### DISCUSSION TITLE
查看器预览PDF的问题

### DISCUSSION THREAD

**Discussion by Lawrence-of-AnKing** [Chaoses-Ib/DirectoryOpus#11] [提问]
目前DO有两个查看器，一个是自带的元插件，另一个是[oracle-outside-in-viewer| Directory Opus](https://chaoses-ib.gitbook.io/directory-opus/wen-jian-liu-lan/cha-kan/cha-kan-qi#oracle-outside-in-viewer)。

两个查看器对于PDF处理都有问题：

1. 元插件：换外屏不重启、用笔记本内屏时，窗口有时变透明。查看很慢，往往要十几二三十秒。
2. Oracle查看器：PDF大于100MB时，直接卡死。但查看很快。

对于元插件，我试着更换预览软件，自带的Microsoft PDF和Edge都可用，但都会有窗口透明问题。

对于Oracle，我没有任何办法，他的问题更严重，一但不小心点到大PDF直接卡死，就算重启预览到也还是重启。

目前只能元插件+取消PDF一种方法。

谁有头绪？

**Comment by Chaoses-Ib** [comment]
传了下 Foxit PDF 和 Sumatra PDF 的预览插件，试试效果？

Foxit PDF: https://github.com/Chaoses-Ib/DirectoryOpus/releases/download/2024-03-04/Foxit.PDF.@Sanhuaitang.v+240818.7z

Sumatra PDF: https://github.com/Chaoses-Ib/DirectoryOpus/releases/download/2024-03-04/Sumatra.PDF.@zldls.v+240818.zip

**Reply by maboloshi** [reply]
wps 自带pdf预览插件也蛮好用的

**Comment by sunyu-ouchn** [comment]
刚好我有PDF预览的问题，也是会卡，今天又研究了下，用Oracle,把他的文件中的vspdf.dll删除，或者是换个位置，元插件里面只留Sumatra PDF，重启下DO，解决了卡的问题，你说的窗口透明的问题，我没有遇到……

**Reply by sunyu-ouchn** [reply]
对了，还需要Oracle优先于元插件
